src/context.rs

```diff
@@ -1374,14 +1374,16 @@ impl IndexMut<BlockOffset> for FrameBlocks {
   }
 }
 
+const MAX_B_SIZE: usize = 512;
```

Comment:
This value is the maximum tile width (4096) divided by the partition context granularity (8x8). Can you define it like that?

Actually also that doesn't really match the left contexts, which are MAX_MIB_SIZE. Either those should be dropped in half or this doubled. Would have to check the partition code to be sure.

Reply:
Sure! the number like that is quite magic indeed  ^^;